code/dictation.py

```diff
@@ -12,26 +12,31 @@
     desc="Look at surrounding text to improve auto-capitalization/spacing in dictation mode. By default, this works by selecting that text & copying it to the clipboard, so it may be slow or fail in some applications.",
 )
 
-@mod.capture(rule="({user.vocabulary} | <word>)")
+@mod.capture(rule="({user.vocabulary} | <user.abbreviated> | <word>)")
 def word(m) -> str:
     """A single word, including user-defined vocabulary."""
     try:
         return m.vocabulary
     except AttributeError:
         return " ".join(actions.dictate.replace_words(actions.dictate.parse_words(m.word)))
 
-@mod.capture(rule="({user.vocabulary} | <phrase>)+")
+@mod.capture(rule="({user.vocabulary} | <user.abbreviated> | <phrase>)+")
 def text(m) -> str:
     """A sequence of words, including user-defined vocabulary."""
     return format_phrase(m)
 
-@mod.capture(rule="({user.vocabulary} | {user.punctuation} | <phrase>)+")
+@mod.capture(rule="({user.vocabulary} | <user.abbreviated> | {user.punctuation} | <phrase>)+")
 def prose(m) -> str:
     """Mixed words and punctuation, auto-spaced & capitalized."""
     text, _state = auto_capitalize(format_phrase(m))
     return text
 
-
+@mod.capture(rule="brief {user.abbreviation}")
+def abbreviated(m) -> str:
+    """A reverse abbreviation inside another command"""
+    return m.abbreviation
+
```

Comment:
We should test whether Dragon performs OK using this definition. If it doesn't, we should make this a contextual override for the capture instead of its global definition, in a context matching `speech.engine: wav2letter`.

Reply:
@wolfmanstout are you still using Dragon and would you be able to take a look?

Reply:
I'm not using Dragon any more regularly. Last I checked it was already really slow. If no one else has Dragon I can try to find some time to test, but I don't have much time these days so it may not be this weekend.

Reply:
Hmm do we know anyone else using Dragon these days?  Maybe we should ping on slack to see if anyone is willing to help out and test this branch?